build/test.ps1

```diff
@@ -57,6 +57,17 @@ if ($Env:ENABLE_QIRRUNTIME -ne "false") {
     Write-Host "Skipping test of qir runtime because ENABLE_QIRRUNTIME variable is set to: $Env:ENABLE_QIRRUNTIME."
 }
 
+
+if ($Env:ENABLE_EXPERIMENTALSIM -ne "false") {
```

Comment:
I missed this in the previous review, but this script should warn if cargo is missing and skip the test like the build script does. That way test.ps1 won't fail for folks who haven't built opensim due to missing cargo install.

Reply:
Sounds good, let me fix that quickly, then.